pkg/install/daemonset.go

```diff
@@ -26,7 +26,7 @@ import (
 
 func DaemonSet(namespace string, opts ...podTemplateOption) *appsv1.DaemonSet {
 	c := &podTemplateConfig{
-		image: "gcr.io/heptio-images/velero:latest",
+		image: "velero/velero:latest",
```

Comment:
kind of unrelated, but we should be using `DefaultImage` here like we do for the Deployment so it's version-tagged by default. I see that in the `velero install` flow we end up passing a `podTemplateOption` to explicitly set the image to a version-tagged one, so this isn't actually a bug in the `velero install` logic, but let's update it anyway.